### ISSUE TITLE
Schedule is incorrect.

### ISSUE THREAD

**Issue by Gryphon324** [jasongdove/ErsatzTV#964]
I'm having issues with one of my channels not properly scheduling titles. I have a channel that is scheduled with only 3 TV show titles, yet Ersatz is scheduling movies and things that are not part of the defined schedule. I had recently made a change to the channel/schedule and assumed that Ersatz somehow cached the scheduling data. So I decided to wait it out for a few days to see if the channel would refresh with the proper scheduled items. After 3 days, the schedule is still incorrect.

![etv1](https://user-images.githubusercontent.com/11334792/190914040-d3995972-6032-418c-adcb-1b216fb0b270.JPG)
![etv2](https://user-images.githubusercontent.com/11334792/190914049-d88f193d-4a6e-41fe-87db-07c1f3861591.JPG)
![etv 3](https://user-images.githubusercontent.com/11334792/190914053-e65f2d6f-566b-4051-aae8-67b532c36dde.JPG)

**Comment by Gryphon324** [comment]
What's interesting is that the channel in question (channel 1) is has scheduled the same movies that are currently scheduled in channels 10, 11 and 12.

**Comment by jasongdove** [comment]
Please confirm the scheduling irregularity using the playout details in ErsatzTV (click on a playout row and another table will appear below). Otherwise I think this is probably https://github.com/jasongdove/ErsatzTV/issues/932

**Comment by Gryphon324** [comment]
Good catch. I checked the playout details and they show that only the three tv shows are being scheduled. I went a step further and looked at the xmltv.xml output and confirmed that only those three shows were being sent to plex.
![evt4](https://user-images.githubusercontent.com/11334792/190930532-d0c27b85-5fc5-4ae5-a51f-b2defc937e4e.JPG)
![etv5](https://user-images.githubusercontent.com/11334792/190930534-a0756d20-9b1b-4984-b4e5-916b26b76d16.JPG)

Obviously plex is not translating it properly. I will have to do more investigation with plex to see what is up. I have downloaded the xmltv.xml file several times but that doesn't seem to affect Plex's lineup.
